src/main/java/ui/view/LinearLayout.java

```diff
@@ -200,7 +204,8 @@ public void onLayout (int width, int height, int x, int y) {
                 }
 
                 if ((this.gravity & Gravity.CENTER) == Gravity.CENTER) {
-                    childY = (int) ((height - totalLength) * 0.5f);
+                    //TODO: fix for when the totalLength is greater than height
+                    //childY = (int) ((height - totalLength) * 0.5f);
```

Comment:
Would it be better to just delete code that isn't in use and add a ticket to fix it?

Reply:
I still want the code, just needs to be added to. commented out as it brakes things as is